### PR TITLE
Feat: article images

### PR DESCRIPTION
**Pull request by sericaia**
Same as https://github.com/newsuk/times-components/pull/213 but to master, as https://github.com/newsuk/times-components/pull/183 is already in!

NOTE: We support only Mobile web, ios and android. Desktop to follow after responsive design

the other styles for desktop were removed on purpose for now

### CODE REVIEW

File: packages/article-image/article-image.js [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+    return null;
+  }
+  return (
+    <View style={styles[`${display}Caption`]}>
+      <Caption text={caption} credits={credits} style={captionStyle[display]} />
+    </View>
+  );
+};
+
+const ArticleImage = ({ ast }) => (
```
Comment (craigbilner) [inline packages/article-image/article-image.js]: This doesn't look right? Should an `ArticleImage` component which AFAICT calculates an aspect ratio and sticks a caption on take an `ast` of images? Surely the article will `renderTrees` and handle images in their renderer?

File: packages/article-image/article-image.js [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+          return null;
+        }
+
+        const source = {
+          uri: url
+        };
+
+        const ratioStyle = getImageRatioBox(ratio);
+        return (
+          <View key={id} style={styles[display]}>
```
Comment (craigbilner) [inline packages/article-image/article-image.js]: ...all the nested views...not great for perf?

File: packages/article-image/article-image.js (hunk 1 above)
Comment (sericaia) [reply packages/article-image/article-image.js]: changing it from ast to `<ArticleImage image={imageDetails} caption={captionDetails} />`

File: packages/article-image/article-image.js (hunk 2 above)
Comment (sericaia) [reply packages/article-image/article-image.js]: They are required due to the paddingBottom trick

File: packages/article-image/article-image.js [hunk 3]
```diff
@@ -0,0 +1,171 @@
...
+      }
+    })
+  }
+});
+
+const getImageRatioBox = ratio => {
+  const tokens = ratio.split(":");
+  const padding = tokens[1] / tokens[0] * 100;
+
+  return StyleSheet.create({
```
Comment (thatportugueseguy) [inline packages/article-image/article-image.js]: Maybe we should move Platform.select to this level, or even more up (return early)?

File: packages/article-image/article-image.js [hunk 4]
```diff
@@ -0,0 +1,171 @@
...
+  return (
+    <View key={url} style={styles[display]}>
+      <View style={styles[`${display}Image`]}>
+        <View style={ratioStyle.ratioBox}>
+          <View style={ratioStyle.ratioBoxInside}>
+            <Image source={source} />
+          </View>
+        </View>
+      </View>
+      {renderCaption(display, caption, credits)}
```
Comment (thatportugueseguy) [inline packages/article-image/article-image.js]: do we really need this function? we could have an if statement instead

Comment (sericaia) [reply packages/article-image/article-image.js]: not going to be fixed as we discussed ;)

**Pull request merged**